src/ucs/sys/sock.h

```diff
@@ -428,6 +428,20 @@ ucs_status_t ucs_sock_ipstr_to_sockaddr(const char *ip_str,
 int ucs_sockaddr_is_known_af(const struct sockaddr *sa);
 
 
+/**
+ * Extract the IP address from a given socket fd.
+ *
+ * @param [in]   fd          Socket fd.
+ * @param [out]  sock_addr   IP address.
+ * @param [out]  addr_len    Length of the IP address.
+ *
+ * @return ip_str if the sock_addr has a valid IP address or 'Invalid address'
+ *         otherwise.
```

Comment:
minor: it returns `UCS_OK` if sock_addr has a valid IP address

Reply:
@brminich  Fixed.